mcs/class/System.Private.CoreLib/System/Enum.cs

```diff
@@ -41,10 +41,17 @@ public override bool Equals (object obj)
 			return DefaultEquals (this, obj);
 		}
 
-		[Intrinsic]
+		[MethodImplAttribute(MethodImplOptions.InternalCall)]
+		private extern bool InternalHasFlag(Enum flags);
+
 		public bool HasFlag (Enum flag)
```

Comment:
This is an JIT intrinsic in mono, so it should still have the [Intrinsic] attribute.

Reply:
It wasn't treated as such and the non-netcore Mono build also uses `InternalHasFlag`.

Reply:
I'll can add it back, but shouldn't it be in mscorlib too?

Reply:
It doesn't matter much, we don't check for it anywhere, its just a marker.